misc/json-to-record-converter/src/test/resources/json/sample_6.json

```diff
@@ -17,6 +17,7 @@
       "position": "Shooting Guard"
     },
     {
+      "sport": null,
```

Comment:
shall we also add a test case where a certain json field(e.g. `sport`) is null within one object and does not exist with an other object (so that the generated record field will be anydata + optional)?